slices/vim-common.yaml

```diff
@@ -0,0 +1,15 @@
+package: vim-common
+
+essential:
+  - vim-common_copyright
+
+slices:
+  configs:
```

Comment:
I think we should name this slice `config` so that it is consistent with the naming scheme present in other SDFs.